File: EjerciciosPracticar JUNIO1/src/Ev2/PracticaU10_11/Ej3.java
package Ev2.PracticaU10_11;

import java.util.HashMap;
import java.util.Scanner;

public class Ej3 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Introduce una frase");
        String frase = sc.nextLine().toLowerCase();
        char c = 0;
        int posicion = 0;
        HashMap<Character, Integer> map = new HashMap<>();

        for (int i = 0; i < frase.length(); i++) {
            char c2 = frase.charAt(i);
            if (map.containsKey(c2)) {
                c = c2;
                posicion = i;
                break;
            } else {
                map.put(c2, i);
            }

        }

        System.out.println("El primer caracter : " + c + " se repite en la posicion : " + (posicion + 1));


    }
}

File: KOTLIN U7/untitled/src/Ej3.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val Scanner = Scanner(System.`in`)
    print("Introduce una palabra : ")
    var palabra = Scanner.nextLine()
    palabra = palabra.lowercase()
    if(palabra == palabra.reversed()){
        print("Es palindromo")
    }else{
        print("no es palindromo")
    }
}